### ISSUE TITLE
Code Table Request - REMOVE examined for parasites

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6225]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

[examined for parasites](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#examined_for_parasites) should be merged into some combination of

* https://arctos.database.museum/info/ctDocumentation.cfm?table=CTATTRIBUTE_TYPE&field=detected#examined_for
* https://arctos.database.museum/info/ctDocumentation.cfm?table=CTATTRIBUTE_TYPE&field=detected#detected
 * https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#not_detected

Migration path and value needs determined; I don't understand these data. 

This will likely involve another code table request for a new value in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctexamined_detected

Data:
[temp_efp.csv.zip](https://github.com/ArctosDB/arctos/files/11383939/temp_efp.csv.zip)

Summary:

```


 guid_prefix | count 
-------------+-------
 MVZ:Mamm    |     1
 DMNS:Mamm   |    20
 UAM:Mamm    |     2
 MSB:Mamm    |  3057
 MSB:Host    | 21800
 UMZM:Mamm   |    66


```

Users:

 @mkoo
 @campmlc
 @amgunderson
 @cjconroy
 @AdrienneRaniszewski
 @acdoll
 @jldunnum
 @jrdemboski
 @msbparasites
 @adhornsby

_**Context**: Describe why this new value is necessary and existing values are not._

We seem to have two ways of doing the same thing.

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

Code table cleanup should always be highest priority.
@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by campmlc** [comment]
I support this for MSB collections. However, we need first to add "parasites" as an option in 
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctexamined_detected

And the examined for attribute will need to be expanded to include the following collections:
Bird
Herp
Inv
Teach

This particular attribute started off as an attribute for the MSB:Para Rausch collection, and as it was all we had, it was used for a very wide range of possible "parasite" types. Most of the detail on who, what, when, and how  was put in collection object remarks for lack of anywhere else. It appears to have been used for other collections now, so it likely has even less consistent usage.  Eventually with resources we may be able to extract more info from the Rausch collection data, but no resources are currently available for this. So adding "parasites" to the "Examined-Detected" attribute table is reasonable. 

@msbparasites

**Comment by dustymc** [comment]
>need first to add "parasites" as an option 

Please file an Issue (or ping me with a definition and I'll do it)

(BUT, apparently some part of the things that get dumped into parasite vials are postmortem arrivals - I wonder if now would be a good time to be more generic/accurate, or if that would just be a distraction and everybody that matters already understands the nature of this stuff??)

**Comment by dustymc** [comment]
I believe this was sufficiently approved by @ArctosDB/arctos-code-table-administrators and I would like to begin work on it TOMORROW.

@campmlc @ccicero @jldunnum please confirm and check some boxes.

**Comment by dustymc** [comment]
fresh data:

[Archive 7.zip](https://github.com/ArctosDB/arctos/files/11520958/Archive.7.zip)

done
